EIPS/eip-common_metrics.md

```diff
@@ -0,0 +1,56 @@
+---
+eip: <to be assigned>
```

Comment:
Please use 2159 and name the file eip-2159.md

Reply:
Done.